README.md

```diff
@@ -8,80 +8,27 @@ The future face of osu!.
 Requirements
 ------------
 
-- A PHP 7.1 (or equivalent) environment
+- A PHP 7.1+ environment
 - MySQL 5.7+
```

Comment:
Not until 5.5 update with [this pull request](https://github.com/laravel/framework/pull/24038) is released.